### PR TITLE
Adding Testing Documentation

### PR DESCRIPTION
**Pull request by jamesbrumpton**
**Overall change:** _Adding testing documentation for testers & non-testers on how to test SIMORGH and PSAMMEAD._

**Code changes:**

- _Adding documentation to explain how to test SIMORGH and PSAMMEAD._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: docs/Testing-Simorgh.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# TESTING FOR ARTICLES
+
+## STORY REFINEMENT
```
Comment (staylos92) [inline docs/Testing-Simorgh.md]: Is it worth adding when you would expect this meeting to take place?

File: docs/Testing-Simorgh.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# TESTING FOR ARTICLES
```
Comment (staylos92) [inline docs/Testing-Simorgh.md:1]: As this document has a lot of info I think adding a contents list with links to the relevant sections would be helpful 👍

File: docs/Testing-Simorgh.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- Once your Local environment has booted up, open a seperate tab and run `npm run lighthouse`
+- Firefox will automatically open. Lighthouse tests will run and a report will be generated in the browser
+
+## PSAMMEAD
+The current testing on Psammead on your Local environment is mainly for checking that styles are loaded as expected and that breakpoints, which can be found on Zeppelin, are as expected. As Psammead is displaying components via an iframe, Articles have agreed that the browser/device testing should take place once each component has been intergrated into Simorgh.
+
+## HOW TO TEST
+Make sure that your version of `latest` is up to date by using `git status` and then, if necessary, `git pull`. If your branch is not up to date when you do a `git pull`. Once your `latest` is up to date, check out the necessary branch. Once the branch has been checked out, make sure to run `npm run install:packages `. Once this has been completed, you can now boot up your Local environment. Once you've ran the command in Terminal, Storybook will open Firefox automatically. Here is the commands required to boot up your Local enviroment.
```
Comment (staylos92) [inline docs/Testing-Simorgh.md]: Super picky..."Here is the *command* required"

**Comment by staylos92** [comment]
In general, are we ok with adding test links to this repo with it being publically accessible?

**Comment by ChrisBAshton** [comment]
Moved to https://github.com/bbc/simorgh-infrastructure/pull/339.

**Pull request closed without merging**